### ISSUE TITLE
[Bug] Return of the ship launch on hosted player exit

### ISSUE THREAD

**Issue by Xenoprophet-Nail** [tsunamayo/Starship-EVO#2979]
### Short Description of the issue:
Yeah it's back, was hosting a server with a friend (cateye13) and he exited his ship and the ship disappeared, it did it several times on us as well.
...

### Steps to reproduce the issue:

...

### Environment
- Starship EVO version: 20w37a
- Graphics Card: Nvidia Geforce 1060 gtx 6gb

### Additional information:
[output_log.txt](https://github.com/tsunamayo/Starship-EVO/files/5244242/output_log.txt)

**Comment by Cateye13** [comment]
Short Description:
Ship Jumping 100-200km away from starting point while exiting from control of ship. 
Caused by either 
a. Exiting the control of the yoke, after light cruising to a area.
b. Exiting the control of the yoke, and exiting the seat, after light cruising to a area.
c. Exiting and then getting back into the ship after light cruising to a area. 

Environment:
Starship EVO version: 20w37a
Graphics card: AMD Ryzen 5 1600 six-Core Processor 8GB

Additional Information
Related Ship file attached, with error log.

[New Compressed (zipped) Folder.zip](https://github.com/tsunamayo/Starship-EVO/files/5244264/New.Compressed.zipped.Folder.zip)

[output_log.txt](https://github.com/tsunamayo/Starship-EVO/files/5244311/output_log.txt)

**Comment by StarrKiss** [comment]
Getting this too lol playing with a friend-
Environment:
 Starship EVO version: 20w37a
 Graphics Card: Nvidia GTX 1070-ti

**Comment by tsunamayo** [comment]
Ok gotcha, sad to see this was never really fixed.... I keep you posted - thanks for letting me know!
Cheers

**Comment by Xenoprophet-Nail** [comment]
Yeah, no problem, it's odd that the issue came back up after being fine for so long

**Comment by tsunamayo** [comment]
I think we just dont have a clear test case yet. I was never able to reproduce it locally, it might be dependent on the ping? Do you know if the other player were located far off? Cheers

**Comment by SovietSponge** [comment]
Yeah I can confirm this bug, I played with xeno a lil bit back and a few of my ships just zoomed to god knows where after I left the seat.
